p2p/server/server_test.go

```diff
@@ -33,7 +32,7 @@ func TestServer(t *testing.T) {
 	}
 	opts := []Opt{
 		WithTimeout(100 * time.Millisecond),
-		WithLog(logtest.New(t)),
+		WithLog(zaptest.NewLogger(t)),
```

Comment:
I just noticed on the other PR that the default level of the `zaptest` logger is `Debug`, while our `logtest` default level is `Info`. So consider decreasing the logging level if this causes too much spam in testlogs 🙂:

```suggestion
		WithLog(zaptest.NewLogger(t, zaptest.Level(zap.InfoLevel))),
```

Reply:
Agree, but we probably should use the same env var as in the custom logger actually via some wrapper fn, as debug logging in the tests is useful sometimes

Reply:
I don't think we need this custom environment variable. We should just switch the `GOTESTUM_FORMAT` setting to `standard-quiet` instead of `standard-verbose`. Then there are is log output unless the test failed, in which case it will print the full (debug) logs for easier investigation.

Then we can also just stick with the default of `DEBUG` rather than passing `zaptest.Level(zap.InfoLevel)` to all tests...

Wdyt?